releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -2,6 +2,9 @@
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
 
+### 5.0.0 - Feb 02, 2022
+* Added deprecation warning to the `SetDirty` API. `ProcessPushNotification` API should be used instead for push-model based configuration refresh. [#301](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/301)
```

Comment:
I think this is where a doc pointer to ProcessPushNotification would come in handy.